junseungbin/ProgrammersList/ProgrammersList.java

```diff
@@ -0,0 +1,86 @@
+package testingfile;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Arrays;
+import java.util.Scanner;
+
+class Solution {
+    public int[] solution(int n, int a, int b, int c, int[] num_list) {
```

Comment:
프로그래머스에서는 solution 메서드의 입력 매개변수가 미리 정해져 있는데
현재 작성하신 메서드와 형태가 달라 실행 시 오류가 발생하는 것 같아요 😢

```suggestion
    public int[] solution(int n, int[] slicer, int[] num_list)
```